### ISSUE TITLE
[inverted colors] action buttons place a black bar across the page

### ISSUE THREAD

**Issue by kfitz** [mlaa/cac-advanced-profiles#78]
Accessibility buttons (at left margin) don't produce expected behavior; the color transformation button places a black bar across the avatar and username, and the text size button actually makes the username smaller.
